common/src/main/java/io/netty/util/ResourceLeakDetector.java

```diff
@@ -97,10 +97,12 @@ static Level parseLevel(String levelStr) {
     }
 
     private static Level level;
+    private static final Level stableLevel;
 
     private static final InternalLogger logger = InternalLoggerFactory.getInstance(ResourceLeakDetector.class);
 
     static {
+        final boolean immutableLevel = SystemPropertyUtil.get("io.netty.immutableResourceLeakDetection") != null;
```

Comment:
I suggest changing the property name and explicitly treating it as a boolean:

```suggestion
        final boolean immutableLevel = Boolean.getBoolean("io.netty.leakDetection.level.immutable");
```

Reply:
but use `SystemPropertyUtil.getBoolean()` instead of `Boolean.getBoolean()`